javascript/javascript1/week2/candy-helper.js

```diff
@@ -0,0 +1,49 @@
+const candies = {
+  Sweet: 0.5,
+  Chocolate: 0.7,
+  Toffee: 1.1,
+  ChewingGum: 0.03,
+};
+
+const buyCandyPrices = [];
+
+function addCandy(candyType, weight) {
+  if (typeof candyType !== 'string' || typeof weight !== 'number') {
+    console.log('Please enter a valid value!');
+  } else {
+    buyCandyPrices.push(weight * candies[candyType]);
+  }
+}
+
+addCandy('Sweet', 20);
```

Comment:
`addCandy("sweet", 20); // Adds the price of 20 grams of sweets to the array boughtCandyPrices`
This should also work